docs/users_guide/landice/test_groups/index.rst

```diff
@@ -15,4 +15,5 @@ in the coming months.
    enthalpy_benchmark
    eismint2
    greenland
+   thwaites
```

Comment:
The corresponding addition is needed in the `docs/developers_guide/landice/test_groups/index.rst`.

Also, so far the tests have been alphabetical.  There's no reason that needs to be the case.  You may want to give them a more logical order if there is one.  That could be something for another PR, perhaps as more test groups get added